templates/_layouts/docs.html

```diff
@@ -48,7 +48,7 @@
             <ul class="p-side-navigation__list">
               <li class="p-side-navigation__item--title"><span class="p-side-navigation__text">Base elements</span></li>
               {{ side_nav_item("/docs/base/code", "Code") }}
-              {{ side_nav_item("/docs/base/forms", "Forms", 'updated') }}
+              {{ side_nav_item("/docs/base/forms", "Forms", 'new') }}
```

Comment:
While editing here could you also remove the ones from other unrelated items from prev version (accordion, cards)